src/objective-c/tests/InteropTestsLocalSSL.m

```diff
@@ -57,4 +57,15 @@ - (void)setUp {
   [super setUp];
 }
 
+- (void)testExceptions {
```

Comment:
I'll add a todo to create a new file. For now I'd like to keep the file structure so we can reuse the coverage information.